src/Answers/Answers.js

```diff
@@ -16,7 +17,7 @@ const Answers = (props) => {
         id={answer}
         onClick={e => props.checkAnswer(e)}
         >
-        {`${answer}`}
+        {decode(answer)}
```

Comment:
This is the most beautiful thing I've ever seen. No more &#!

Reply:
Package definitely worth it...still unsure why the occasional code slip past though